src/components/transactions/LiTransaction.vue

```diff
@@ -63,6 +63,10 @@ export default {
     fees: {
       type: Object,
       required: true
+    },
+    hideData: {
```

Comment:
Maybe something more descriptive like: `preSend`?

Reply:
Or `hideBlockData`

Reply:
maybe `hideRightSide`

Reply:
What's the purpose on this scenario to hide the fees? Can't we jsut check the fees prop?

Reply:
the fees are visible below. we're also hiding the block number and datetime.

<img width="573" alt="Screen Shot 2019-06-21 at 8 44 18 AM" src="https://user-images.githubusercontent.com/6021933/59923212-da9ace00-9400-11e9-9da7-7a97a4f46292.png">